tests/cucumber/e2e/pageObjects/LoginPage.js

```diff
@@ -0,0 +1,29 @@
+require('dotenv').config()
+
+class LoginPage{
+    constructor(){
+        this.homepageSelector = "//div[@class='container-lg']"
+        this.usernameSelector = "//input[@id='validationDefault02']"
+        this.passwordSelector = "//input[@id='validationDefault03']"
+        this.loginButtonSelector = "//button[@class='btn btn-primary']"
+        this.loginPageSelector = "//a[@class='nav-link dropdown-toggle text-primary']"
+        this.errorTextSelector = "//div[@class='alert alert-danger']"
+    }
+    
+    async navigateLogin(){
+        await page.goto(process.env.APP_URL+"login")
+        return await page.locator(this.homepageSelector)
+    }
```

Comment:
```suggestion
    async navigateLogin(){
        await page.goto(process.env.APP_URL+"login")
        return await page.locator(this.homepageSelector)
    }
```
Is this function used to navigate the login page? What is the purpose of return statement in this function?